utils/python/CIME/build.py

```diff
@@ -511,7 +511,7 @@ def build_libraries(case, exeroot, caseroot, cimeroot, libroot, mpilib, lid, mac
         sharedpath = os.environ["SHAREDPATH"]
         bldroot = os.path.join(sharedpath, case.get_value("COMP_INTERFACE"), esmfdir, "clm","obj" )
         libroot = os.path.join(sharedpath, case.get_value("COMP_INTERFACE"), esmfdir, "lib")
-        incroot = os.path.join(sharedpath,"include")
+        incroot = os.path.join(sharedpath, case.get_value("COMP_INTERFACE"), esmfdir, "include")
```

Comment:
There was a conflict between the dlnd lnd_comp_mct.mod and the clm lnd_comp_mct.mod - moving the location of the clm include directory solves this problem.